src/deepEqual.js

```diff
@@ -24,7 +24,63 @@
  */
 
 function deepEqual(a, b) {
```

Comment:
overall this solution doesn't really work. 

you need to run through each property of object A and B and compare them. 
if properties are pimitives - just compare by values. 
if properties are objects - compare recursively. 

hope that helps.